vhdbuilder/packer/install-dependencies.sh

```diff
@@ -161,14 +161,24 @@ echo "VHD will be built with containerd as the container runtime"
 updateAptWithMicrosoftPkg
 containerd_manifest="$(jq .containerd manifest.json)" || exit $?
 
-installed_version="$(echo ${containerd_manifest} | jq -r '.edge')"
+containerd_installed_version="$(echo ${containerd_manifest} | jq -r '.edge')"
```

Comment:
nit: can we improve the naming here? i know it was broken from before, but I wonder how we can make it better?
for instance, that variable signifies the containerd version that is going to be installed, so maybe we could align a bit there?

Reply:
Good point. Actually I also had a question. In the manifest, it has the key "runc.installed", so it should have a better name too, right? I can modify it along with the change here. The best name versions are taken in runc but it's not actually used.